charts/README.md

```diff
@@ -14,6 +14,13 @@ $ helm package azuredisk-csi-driver
 $ helm install azuredisk-csi-driver azuredisk-csi-driver-latest.tgz --namespace kube-system
 ```
   
+## Install latest AzureDisk CSI Driver on Azure Stack via `helm install`
+
+```console
+$ cd $GOPATH/src/sigs.k8s.io/azuredisk-csi-driver/
+$ helm install azuredisk-csi-driver ./charts/latest/azuredisk-csi-driver --namespace kube-system --set cloud.isAzureStack=true
```

Comment:
I am thinking what about `--set cloud=AzureStack`, we also need to support sovereign cloud, e.g. `AzureChina`, it need to set different container registry other than `mcr.microsoft.com`, we could do some customization based on cloud name, `--set cloud=AzureStack` would be more generic.

Reply:
Make sense. Did the change.